{
    "error": {
        "code": 400, 
        "errors": [
            {
                "domain": "global", 
                "location": "query", 
                "locationType": "other", 
                "message": "Encountered \" \"GROUP\" \"GROUP \"\" at line 40, column 5.\nWas expecting:\n    \")\" ...\n    ", 
                "reason": "invalidQuery"
            }
        ], 
        "message": "Encountered \" \"GROUP\" \"GROUP \"\" at line 40, column 5.\nWas expecting:\n    \")\" ...\n    "
    }
}